src/ds/d400/d400-private.cpp

```diff
@@ -317,6 +317,19 @@ namespace librealsense
             }
             return false;
         }
-
+        
+        notification d400_notification_decoder::decode( int value )
+        {
+            if( ds::d400_fw_error_report.find( static_cast< uint8_t >( value ) ) != ds::d400_fw_error_report.end() )
```

Comment:
Please make sure it still works on d400 somehow

Reply:
Verified using dedicated HWMC